docker-compose.yml

```diff
@@ -0,0 +1,102 @@
+version: "3.4"
+services:
+  clickhouse:
+    image: yandex/clickhouse-server:20-alpine
+    ports:
+      - 8123:8123
+      - 9000:9000
+  impala:
+    depends_on:
+      - postgres
+    environment:
+      PGPASSWORD: postgres
+    healthcheck:
+      interval: 30s
+      retries: 20
+      test:
+        - CMD-SHELL
+        - nc -z 127.0.0.1 21050 && nc -z 127.0.0.1 50070
+      timeout: 10s
+    hostname: localhost
+    image: ibisproject/impala:latest
+    ports:
+      - 9020:9020
+      - 50070:50070
+      - 50075:50075
+      - 8020:8020
+      - 8042:8042
+      - 9083:9083
+      - 21000:21000
+      - 21050:21050
+      - 25000:25000
+      - 25010:25010
+      - 25020:25020
+  postgres:
```

Comment:
Rename it to `impala-postgres`?

Reply:
It was easier to change postgres to avoid dealing with needing a network alias. The impala container specifically requires `postgres:5432`.